<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>391</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>391</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Element length correction setup</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>8</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Element length correction setup</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>400</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QTabWidget" name="tabWidget_element_length_correction">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>42</y>
      <width>389</width>
      <height>301</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>227</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>400</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_model">
     <attribute name="title">
      <string>Model</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_confirm">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>198</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>22</y>
        <width>327</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Select the element length correction type:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>80</y>
        <width>131</width>
        <height>87</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioButton_loop">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Loop</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButton_expansion">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Expansion</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_side_branch">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Side branch</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_remove">
     <attribute name="title">
      <string>Remove</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_remove">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>6</y>
        <width>369</width>
        <height>257</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>By group</string>
       </attribute>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>4</y>
          <width>347</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Select a group to remove the length correction:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QTreeWidget" name="treeWidget_length_correction_groups">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>36</y>
          <width>341</width>
          <height>150</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <column>
         <property name="text">
          <string>Group</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Elements</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="pushButton_remove_by_group_confirm">
        <property name="geometry">
         <rect>
          <x>144</x>
          <y>192</y>
          <width>80</width>
          <height>29</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>29</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_get_information">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>150</y>
          <width>80</width>
          <height>29</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>29</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>See details</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>By element</string>
       </attribute>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>4</y>
          <width>353</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Select a element to remove the length correction:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QTreeWidget" name="treeWidget_length_correction_elements">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>36</y>
          <width>280</width>
          <height>150</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <column>
         <property name="text">
          <string>Elements</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="pushButton_remove_by_elements_confirm">
        <property name="geometry">
         <rect>
          <x>144</x>
          <y>192</y>
          <width>80</width>
          <height>29</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>29</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_selection">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>10</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Elements IDs:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_elementID">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>10</y>
      <width>170</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_confirm">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>8</y>
      <width>80</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>29</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>29</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(125, 255, 245);</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
